android/src/main/java/com/opentokreactnative/utils/Utils.java

```diff
@@ -100,6 +101,25 @@ public static List<IceServer> sanitizeIceServer(ReadableArray serverList) {
         return iceServers;
     }
 
+    public static ArrayList<VideoTransformer> sanitizeVideoTransformerList(PublisherKit publisher, ReadableArray transformerList) {
```

Comment:
otkit internally does this sort of checks. Trying to set Background Replacament should not lead to anything. If it leads to a crash or something similar, that is a bug on otkit